omod-2.4/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/UserController1_8Test.java

```diff
@@ -243,9 +243,6 @@ public void getUser_shouldGetAFullRepresentationOfAPatient() throws Exception {
 		
 		Assert.assertNotNull(result);
 		assertEquals(getUuid(), PropertyUtils.getProperty(result, "uuid"));
-		
-		Assert.assertNotNull(PropertyUtils.getProperty(result, "secretQuestion"));
```

Comment:
Did you intentionally remove these?

Reply:
Yes, no more secretQuestion in representation.